actions/channel_actions.test.js

```diff
@@ -101,6 +101,7 @@ jest.mock('mattermost-redux/actions/channels', () => ({
             }],
         };
     },
+    addChannelMember: (...args) => ({type: 'MOCK_ADD_USER_TO_CHANNEL', args}),
```

Comment:
nit: Please change to `MOCK_ADD_CHANNEL_MEMBER` - easier to follow as mock of `addChannelMember`.